warehouse/warehouse.go

```diff
@@ -2090,6 +2130,11 @@ func Start(ctx context.Context, app app.App) error {
 		asyncWh = jobs.InitWarehouseJobsAPI(ctx, dbHandle, &notifier)
 		jobs.WithConfig(asyncWh, config.Default)
 
+		// Add code here to associate ssh keypair with destination data.
+		g.Go(misc.WithBugsnagForWarehouse(func() error {
+			return nil
+		}))
+
```

Comment:
```suggestion

```

Do we need this ?

Reply:
No we don't, I'll remove this remanent of past.